package.json

```
@@ -5,6 +5,7 @@
     "name": "PagoPA"
   },
   "private": false,
+  "license": "MIT",
```

```suggestion
  "license": "SEE LICENSE IN LICENSE"
```

Since there is a custom LICENSE file, you should link the license in that way, [as suggested by npm](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#license)